config/v1/config.go

```diff
@@ -15,9 +15,9 @@
 package v1
 
 import (
-	"errors"
 	"reflect"
 
+	"github.com/coreos/ignition/config/errors"
```

Comment:
Combining the commit that adds the errors with the various commits that switch over to using them would make ordering tricky, since the commit that switches over `config/v1` to the errors depends on the work that makes `config/v1` use `config/util` for error handling, and that logic in `config/util` needs those errors.

I could squash together the commits across all the `config/v*` packages that switch over to using the new errors, if you think that would make this cleaner.

Reply:
Personally I think it's fine either way (leaving it or changing as you suggested), unless someone else has any other opinions on this.

Reply:
I'll leave it then